SETUP.md

```diff
@@ -17,32 +17,18 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
      yarn --version  # --> 1.22.5
      ```
 
-2. If using SSH (recommended): `git clone git@github.com:code-dot-org/code-dot-org.git` , if using HTTPS: `git clone https://github.com/code-dot-org/code-dot-org.git`
+1. If using SSH (recommended): `git clone git@github.com:code-dot-org/code-dot-org.git` , if using HTTPS: `git clone https://github.com/code-dot-org/code-dot-org.git`
 
-3. `cd code-dot-org`
+1. `cd code-dot-org`
 
-4. For Apple Silicon (M1), make these changes to your [Gemfile.lock](Gemfile.lock) file to switch from `libv8` to `libv8-node` and upgrade `mini_racer`. These changes should not be committed and will unfortunately clutter your `git status`.
+1. `gem install bundler -v 1.17.3`
 
-   ```text
-   ...
-   libv8-node (15.14.0.0)
-   ...
-   mini_racer (0.4.0)
-     libv8-node (~> 15.14.0.0)
-   ...
-   ```
-
-5. `gem install bundler -v 1.17.3`
-
-6. `export LIBRARY_PATH=$LIBRARY_PATH:/opt/homebrew/opt/openssl/lib/`
-
-7. `rbenv rehash`
+1. `rbenv rehash`
 
-8. `bundle install`
+1. `bundle install`
```

Comment:
pretty sure we can't run `bundle install` on M1 without the lockfile changes.

However, I've got another PR that will hopefully remove this workaround.

Reply:
Right, but the directions for the lockfile changes are in the section referenced by this earlier step:

    1. Install OS-specific prerequisites
       - See the appropriate section below: [OS X](#os-x-monterey---including-apple-silicon-m1), [Ubuntu](#ubuntu-1804-download-iso), [Windows](#windows)

Reply:
oh, i missed one of the 'expand' sections in the diff and it was hiding the M1 heading 🤦

Reply:
Oh, I should have also mentioned:

Some of the instructions I moved, but some of them I went to move and realized they already existed both in the place I was moving them from and the place I was moving them to, so I just removed them.